mantidproject/mantid
# Indirect Data Analysis I(Q, t) Fit Plot Diff in miniplot

Fixes #13983

I(Q,t) Fit should now plot the Diff as well as the calculated fit in the mini plot after fit is run. The guess should also be removed from the plot.
# To Test
- Open I(Q, t) Fit (Interfaces > Indirect > Data Analysis > I(Q, t) Fit)
- Load in run irs26176_graphite002_iqt.nxs available from `\\olympic\Babylon5\Public\ElliotOram\DataAnalysis\Iqt`
- Set EndX to be within the data range (0.25 should be fine)
- Ensure the plot Guess option is checked and the curve (green line) is on the mini plot
- Run the algorithm
- Ensure that Fit (red) and Diff (blue) curves are now on the plot. 
- Ensure that the guess (green) is no longer on the mini plot
